src/rgw/rgw_sal_motr.h

```diff
@@ -1133,7 +1145,7 @@ class MotrStore : public Store {
     int delete_access_key(const DoutPrefixProvider *dpp, optional_yield y, std::string access_key);
     int store_email_info(const DoutPrefixProvider *dpp, optional_yield y, MotrEmailInfo& email_info);
 
-    int init_metadata_cache(const DoutPrefixProvider *dpp, CephContext *cct, bool use_cache);
+    int init_metadata_cache(const DoutPrefixProvider *dpp, CephContext *cct);
```

Comment:
@sumedhak27 : why "bool use_cache" is removed?

Reply:
line 990., introduced a bool for all the configs.
and MotrStore::initialize() will take care of calling the required methods based on the configuration.